digid_eherkenning/saml2/eherkenning.py

```diff
@@ -420,6 +421,15 @@ class eHerkenningClient(BaseSaml2Client):
     cache_key_prefix = "eherkenning"
     cache_timeout = 60 * 60  # 1 hour
 
+    def __init__(
+        self,
+        *args,
+        loa: Optional[AssuranceLevels] = None,
```

Comment:
possibly we can keep the underlying type consistent to strings:

```suggestion
        loa: Union[AssuranceLevels, Literal[""]] = "",
```

This should not affect the `self.loa or ...` check on L484

Reply:
Hmmm. That suggests it's a `str` not a `Maybe AssuranceLevels`.